x/mongo/driver/operation.go

```diff
@@ -268,7 +271,13 @@ type Operation struct {
 	// has more documents than can fit in a single command. This should only be specified for
 	// commands that are batch compatible. For more information, please refer to the definition of
 	// Batches.
-	Batches *Batches
+	Batches interface {
```

Comment:
Suggest internalizing this interface so that we can create a compile check for `mongo.modelBatches` and `x/mongo.Batches`.